components/Comments/CommentForm.vue

```diff
@@ -40,14 +40,25 @@
       },
       replyComment: {
         type: Object
+      },
+      isCommentFormOfContribution: {
+        type: Boolean,
+        default: false
+      },
+      depth: {
+        type: Number,
+        default: 0
       }
     },
     data () {
       return {
         isLoading: false,
+        isExecuted: false,
+        hasSubmitted: false,
         form: {
           content: '',
-          contributionId: null
+          contributionId: null,
+          parentCommentId: ''
```

Comment:
Use null as default value instead of empty string

Reply:
done